recipes/termcap/all/patches/0001-msvc.patch

```diff
@@ -0,0 +1,193 @@
+diff --git a/termcap.h b/termcap.h
```

Comment:
This patch might be too much to make the library shared in MSVC. I am not sure if we should go that far

Reply:
The effort is done, working and amortized as of know.
So no problem imho.

Reply:
let's keep it, as it's already done